internal/pipe/brew/template.go

```diff
@@ -35,17 +35,17 @@ class {{ .Name }} < Formula
   if OS.mac?
     {{- if .MacOS.DownloadURL }}
     url "{{ .MacOS.DownloadURL }}"
+	{{- if .DownloadStrategy }}, :using => {{ .DownloadStrategy }}{{- end }}
     sha256 "{{ .MacOS.SHA256 }}"
     {{- end }}
   elsif OS.linux?
     {{- if .Linux.DownloadURL }}
     url "{{ .Linux.DownloadURL }}"
+	{{- if .DownloadStrategy }}, :using => {{ .DownloadStrategy }}{{- end }}
```

Comment:
I believe this will generate this:

```ruby
url "https://github.com/test/test/releases/download/v1.0.1/bin.tar.gz"
, :using => GitHubPrivateRepositoryReleaseDownloadStrategy
```

Instead of this:

```ruby
url "https://github.com/test/test/releases/download/v1.0.1/bin.tar.gz", :using => GitHubPrivateRepositoryReleaseDownloadStrategy
```

Reply:
not really, its a `{{-`, which removes whitespaces...

tests are also passing with the correct golden files: https://github.com/goreleaser/goreleaser/pull/1048/files#diff-af67cb526fba8f47c4572ec37fcc6975R8

Reply:
@caarlos0 you are right. My bad.